Core/src/Propagator/SympyStepper.cpp

```diff
@@ -77,6 +77,12 @@ SympyStepper::boundState(
       state.pathAccumulated, freeToBoundCorrection);
 }
 
+bool SympyStepper::prepareCurvilinearState(State& state) const {
+  // TODO implement like in EigenStepper
```

Comment:
TODO?

Reply:
I realized that this is not implemented in the sympy stepper yet. I will try to follow up with this in another PR